src/screens/AccountDetails.js

```diff
@@ -33,36 +30,26 @@ import {
 	navigateToLegacyAccountList
 } from '../util/navigationHelpers';
 import fontStyles from '../fontStyles';
+import UnknownAccountWarning from '../components/UnknownAccountWarning';
+import { withAccountStore } from '../util/HOC';
 
-export default class AccountDetails extends React.PureComponent {
```

Comment:
It should not be a `PureComponent`, since the props of the selected account like `account.name` changes even if all the props keep the same value.